modules/op-installing-and-deploying-tekton-hub-on-an-openshift-cluster.adoc

```diff
@@ -3,104 +3,18 @@
 // *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
 
 :_content-type: PROCEDURE
-[id="op-installing-and-deploying-tekton-hub-on-an-openshift-cluster_{context}"]
+[id="installing-and-deploying-tekton-hub-on-an-openshift-cluster_{context}"]
 = Installing and deploying {tekton-hub} on a {product-title} cluster
 
 [role="_abstract"]
 {tekton-hub} is an optional component; cluster administrators cannot install it using the `TektonConfig` custom resource (CR). To install and manage {tekton-hub}, use the `TektonHub` CR.
 
-[NOTE]
-====
-If you are using Github Enterprise or Gitlab Enterprise, install and deploy {tekton-hub} in the same network as the enterprise server. For example, if the enterprise server is running behind a VPN, deploy {tekton-hub} on a cluster that is also behind the VPN.
-====
-
-[discrete]
-.Prerequisites
-* Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
-
-[discrete]
-.Procedure
-
-. Create a fork of the link:https://github.com/tektoncd/hub[Tekton Hub] repository.
+You can install Tekton Hub on your cluster using two modes:
 
-. Clone the forked repository. 
+* Automatic installation, _without_ login authorization and ratings for Hub artifacts
```

Comment:
I won't say this as `Automatic Installation` and `Manual Installation` rather can we say it as just `Without Login and Rating` and `With Login and Rating` ?